xml/operations-maintenance-database_maintenance.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<?xml-stylesheet href="urn:x-suse:xslt:profiling:docbook51-profile.xsl"
+ type="text/xml"
+ title="Profiling step"?>
+<!DOCTYPE section [
+ <!ENTITY % entities SYSTEM "entity-decl.ent"> %entities;
+]>
+<section xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="database-maintenance">
+ <title>Database Maintenance</title>
+  <para>
+  As part of Database maintenance for your cloud. See sections below for cleanup commands:
```

Comment:
I would remove the word "sections" and just have this read "See below for cleanup commands:"

Reply:
I have removed